velox/functions/prestosql/aggregates/MinMaxAggregates.cpp

```diff
@@ -90,6 +94,12 @@ class MinMaxAggregate : public SimpleNumericAggregate<T, T, T> {
   }
 };
 
+template <>
+inline int32_t MinMaxAggregate<UnscaledLongDecimal>::accumulatorAlignmentSize()
```

Comment:
Please can you add a comment about what this represents and why its calculated so ? Also why is UnscaledShortDecimal not similarly overloaded ?

Reply:
Copied comment from SumAggregate.h. It is probably coz UnscaledShortDecimal data may get auto byte aligned as it is 64-bit long.